### PR TITLE
Fixes #20304: Improve instructions in default vhost

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/20304

### CODE REVIEW

File: relay/sources/apache/rudder-vhost.conf [hunk 1]
```diff
@@ -73,3 +95,54 @@
...
+#   <Location "/reports">
+#     Require all granted
+#   </Location>
+#   <Location "/uuid">
+#     Require all granted
+#   </Location>
+#   <Location "/rudder/relay-api">
+#     Require all granted
+#   </Location>
+#
```
[the remark this answers is not in the document]
Comment (amousset) [reply relay/sources/apache/rudder-vhost.conf:129]: nodes shouldn't require access to the public API, which cases are you thinking about?

[the remark this answers is not in the document]
Comment (amousset) [reply relay/sources/apache/rudder-vhost.conf:129]: (actually preventing Web and API access to the nodes is one explicit goal in the related support case)

[the remark this answers is not in the document]
Comment (ncharles) [reply relay/sources/apache/rudder-vhost.conf:129]: ha, it was in the previous virtualhost. my bad

**Comment by amousset** [comment]
Have you tested this patch @ncharles ?

**Comment by ncharles** [comment]
yes, but the <Location "/"> in l 108 needs to be removed, as it prevent all connexions

**Comment by ncharles** [comment]
> Then it does not work as expected as it would not block Web/UI for nodes and nodes endpoints on Web/API.
> 
> I'll test it and try to improve the configuration.

ha, my bad, it was in the previous version
it works with the "/" actually, but i needed to use a different set of ip for that

**Pull request closed without merging**